card/lib/card/model/save_helper.rb

```diff
@@ -83,6 +83,29 @@ def ensure_card! name_or_args, content_or_args=nil
         end
       end
 
+      def ensure_trait name, codename, args
+        ensure_card name, codename: codename
+        args.each do |k, v|
+          send "ensure_trait_#{k}", name, v
```

Comment:
Might be good to protect this with respond_to? and do something more sensible when the params are wrong. Is this just used in migrations? Maybe throwing an error is fine.